src/common/file_op/tests/unit/tests/test_file_op.c

```diff
@@ -1,976 +0,0 @@
-/*
- * Copyright (C) 2015, Wazuh Inc.
- *
- * This program is free software; you can redistribute it
- * and/or modify it under the terms of the GNU General Public
- * License (version 2) as published by the FSF - Free Software
- * Foundation.
- */
-
-#include <stdarg.h>
-#include <stddef.h>
-#include <setjmp.h>
-#include <cmocka.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include "../headers/defs.h"
-#include "../headers/file_op.h"
-#include "../error_messages/error_messages.h"
-#include "../wrappers/common.h"
-#include "../wrappers/libc/stdlib_wrappers.h"
-#include "../wrappers/posix/stat_wrappers.h"
-#include "../wrappers/posix/unistd_wrappers.h"
-#include "../wrappers/wazuh/shared/debug_op_wrappers.h"
-#include "../wrappers/wazuh/shared/file_op_wrappers.h"
-#include "../wrappers/externals/zlib/zlib_wrappers.h"
-
-/* setups/teardowns */
-static int setup_group(void **state) {
-    test_mode = 1;
-    return 0;
-}
-
-static int teardown_group(void **state) {
-    test_mode = 0;
-    return 0;
-}
-
-#ifdef TEST_WINAGENT
-
-#define N_PATHS 5
-
-static void expect_find_first_file(const char *file_path, const char *name, DWORD attrs, HANDLE handle) {
```

Comment:
we should leave a TODO stating that the tests should be replaced or maybe opening an issue with a migration to a library that shouldn't need them.